File: CS/Tests/ActionTests.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Default;
using NUnit.Framework;

namespace Tests {
    public class ActionTests : ODataTestsBase {

        [Test]
        public void FunctionTest() {
            Container container = GetODataContainer();
            decimal sales2017 = container.TotalSalesByYear(2017).GetValue();
            decimal sales2018 = container.TotalSalesByYear(2018).GetValue();

            Assert.AreEqual(0, sales2017);
            Assert.AreEqual(3501.55m, sales2018);
        }
    }
}

File: VB/Tests/ActionTests.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports [Default]
Imports NUnit.Framework

Namespace Tests
    Public Class ActionTests
        Inherits ODataTestsBase

        <Test> _
        Public Sub FunctionTest()
            Dim container As Container = GetODataContainer()
            Dim sales2017 As Decimal = container.TotalSalesByYear(2017).GetValue()
            Dim sales2018 As Decimal = container.TotalSalesByYear(2018).GetValue()

            Assert.AreEqual(0, sales2017)
            Assert.AreEqual(3501.55D, sales2018)
        End Sub
    End Class
End Namespace
